packages/hdwallet-native/src/cosmos.ts

```diff
@@ -0,0 +1,108 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+
+import { addressNListToBIP32, CosmosSignTx, CosmosSignedTx } from "@shapeshiftoss/hdwallet-core";
```

Comment:
We are already importing core as well. can just use `core.addressNListToBIP32` etc